<?xml version="1.0"?>
<launch>
  <!-- start node and load parameters -->
  <group ns="gdx">
    <!-- Run the godirect_publisher node and set config as private parameters -->
    <node pkg="godirect_ros" type="store_calibrating_experiment" name="store_calibrating_experiment"
      output="screen">
      <rosparam command="load"
        file="$(find godirect_ros)/config/calibrating_grip_experiment.yaml"/>
    </node>
  </group>
</launch>